### PR TITLE
Update events queue

### PR DESCRIPTION
**Pull request by mattj-io**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.

### CODE REVIEW

File: src/events.json [hunk 1]
```diff
@@ -1001,6 +1001,12 @@
       "url": "https://ossna18.sched.com/event/FANd?iframe=no"
   },
 
+  { "title": "5th DC/OS and Mesos Meetup",
+      "image": "/assets/images/events/Hangzhou.jpg",
+      "date": "2018-09-15",
+      "url": "http://www.huodongxing.com/event/7451680445500"
+  },
+
```
Comment (pleia2) [inline src/events.json]: While you're in here, do you want to add this one as well on September 20th?

https://www.meetup.com/Bay-Area-MLflow/events/254124344/

File: src/events.json [hunk 2]
```diff
@@ -1025,6 +1031,18 @@
       "url": "https://ohiolinux.org/elizabeth-krumbach-joseph-to-keynote-olf2018/"
   },
 
+  { "title": "Workshop: Building and Operating an OSS Data Science Platform (OSS Summit)",
+      "image": "/assets/images/events/Edinburgh.jpg",
+      "date": "2018-10-21",
+      "url": "https://events.linuxfoundation.org/events/open-source-summit-europe-2018/program/schedule/"
```
Comment (pleia2) [inline src/events.json]: That link just goes to the main schedule, I think you want:

https://osseu18.sched.com/event/Fx5S/workshop-building-and-operating-an-oss-data-science-platform-jorg-schad-mesosphere

File: src/events.json [hunk 3]
```diff
@@ -1025,6 +1031,18 @@
...
+  },
+  
+  { "title": "Keynote: Kubeflow++ - Building and Operating an OSS Data Science Platform (OSS Summit)",
+      "image": "/assets/images/events/Edinburgh.jpg",
+      "date": "2018-10-22",
+      "url": "https://events.linuxfoundation.org/events/open-source-summit-europe-2018/program/schedule/"
```
Comment (pleia2) [inline src/events.json]: Same with this one: https://osseu18.sched.com/event/FxWs/kubeflow-building-and-operating-a-oss-data-science-platform-jorg-schad-mesosphere

File: src/events.json [hunk 4]
```diff
@@ -1049,12 +1067,24 @@
...
   { "title": "Deep Learning Pipelines Anywhere Using Kubeflow (KubeCon China)",
       "image": "/assets/images/events/Shanghai.jpg",
       "date": "2018-11-15",
       "url": "https://kccncchina2018english.sched.com/event/FuJt?iframe=no"
   },
 
+  { "title": "Batch to pipelines (ITNext Summit)",
+      "image": "/assets/images/events/Amsterdam.jpg",
+      "date": "2018-11-14",
+      "url": "https://www.itnextsummit.com/"
```
Comment (pleia2) [inline src/events.json]: This one is a direct link to you profile+talk: https://www.itnextsummit.com/speakers/matt-jarvis/

**Comment by mattj-io** [comment]
@pleia2 changes as requested

**Pull request merged**